### ISSUE TITLE
[Enhancement]: expand drag n' drop to homepage/library page

### ISSUE THREAD

**Issue by sevenlayercookie** [advplyr/audiobookshelf#2612]
### Describe the feature/enhancement

Makes sense to me to be able to drag n' drop an audio file anywhere on any page in audiobookshelf window. I don't think it should be limited to only drag n' drop after clicking on the upload button

**Comment by nichwall** [comment]
So dragging and dropping should open the upload page?

**Comment by sevenlayercookie** [comment]
Yes with the dropped file now selected and ready to enter metadata etc.
